### DISCUSSION TITLE
Where is turbopack ?

### DISCUSSION THREAD

**Discussion by firmwired** [vercel/turborepo#9577] [Help] [answered]
### Summary

I've been searching for turbopack source code and i haven't find it yet, i know it is based on turbo but where is it in this repo ?

### Example

where is turbopack in this repo ?

**Comment by joevingracien** [comment, marked as answer]
It has been moved to the next.js repo to iterate faster afaik: https://github.com/vercel/next.js/tree/canary/turbopack
